probe-dictionary/explore.js

```diff
@@ -967,7 +967,15 @@ function getMeasurementCountsPerVersion() {
           // If so, increase the count.
           if (recording) {
             let k = recording.optout ? "optout" : "optin";
-            data[k] += 1;
+            // For release channel show only "optout" probes
+            if (channel === "release") {
+              if (k === "optout") {
+                data[k] += 1;
+              }  
+            }
+            else {
```

Comment:
I think we can bring this down from 2 `if` and 1 `else` to one single `if` statements if we pick the right logical expression.

Reply:
Yes will do it. I just wanted to confirm my approach...